### ISSUE TITLE
Question about the metric used by losses

### ISSUE THREAD

**Issue by carlok** [alankbi/detecto#109]
Hi and thank you very much for your work.
model.fit returns an array of losses but I'm not sure which metric is used to measure the error between two rectangles (ground truth and the rectangle predicted by the model). Is it IoU?
Is there a way, for a trained model, to get the IoU for a given testing image?
At the moment, I read the API but I didn't read the internal code of detecto.
Thanks again.

**Comment by alankbi** [comment]
At the moment, there isn't an inbuilt way to calculate the IoU - you would likely have to take the output of the model and write your own function to compare it against the ground truth bounding boxes. 

The losses returned during the fitting process are simply what the underlying PyTorch model returns when run on the given validation dataset - according to the discussion [here](https://stackoverflow.com/questions/60339336/validation-loss-for-pytorch-faster-rcnn) and [here](https://discuss.pytorch.org/t/compute-validation-loss-for-faster-rcnn/62333/9), you probably won't get any useful meaning from the exact values themselves, but more so they can be helpful to monitor that the overall trend is good and decreasing.
